File: ts/test/fixtures/samples/sample.ts
export function helper(): number {
  return 1;
}

export class Service {
  run(): void {}
}

export abstract class Base {
  abstract handle(): void;
}

export interface Shape {
  area(): number;
}

export type ID = string;

export enum Color {
  Red,
  Green,
}

const version = "1.0";

File: ts/test/fixtures/samples/sample.cs
using System;

namespace Demo
{
    public delegate void Handler();

    public interface IShape
    {
        double Area();
    }

    public enum Color { Red, Green }

    public struct Point
    {
        public int X;
    }

    public record Person(string Name);

    public class Service
    {
        public Service() {}

        public int Value { get; set; }

        public void Run() {}
    }
}
